pabi_bank_statement_reconcile/models/pabi_bank_statement.py

```diff
@@ -17,6 +22,10 @@ class PABIBankStatement(models.Model):
         string='Import File (*.csv)',
```

Comment:
@Jakkrich this field, if not used, you can remove it.
From user perspective, we don't need CSV import anymore.